src/main/java/hellfirepvp/astralsorcery/client/registry/RegistryRenderTypes.java

```diff
@@ -186,6 +190,168 @@ private static void initEffects() {
                         .build());
     }
 
+    private static void initConstellationTypes() {
+        CONSTELLATION_DISCIDIA_BACKGROUND = createType("discidia_backgrrund", DefaultVertexFormats.POSITION_COLOR_TEX,
```

Comment:
This does need to be redone.
Consider doing a method that takes the differences as parameters.
For ResourceLocations with "astralsorcery:..." please use AstralSorcery.key(path)
Also, "backgrrund" :P